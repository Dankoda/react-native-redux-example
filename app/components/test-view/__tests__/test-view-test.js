// @flow

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TestView from '../test-view';

describe('Test View component', () => {
  it('should render with a text prop', () => {
    const props = {
      text: 'This is test text',
    };

    const tree = toJson(
      shallow(<TestView {...props} />),
    );

    expect(tree).toMatchSnapshot();
  });
});
