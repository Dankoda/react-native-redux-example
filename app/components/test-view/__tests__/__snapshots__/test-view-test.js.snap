exports[`Test View component should render with a text prop 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    This is test text
  </Text>
</View>
`;
